### ISSUE TITLE
Run

### ISSUE THREAD

**Issue by Alcala** [AgnaldoJaws/credentialfaces-PHP#1]
Desculpe o incômodo e a forma a qual estou entrando em contato, mas como faço para rodar esse script?

**Comment by AgnaldoJaws** [comment]
opa blz ?? então as configs de banco de dados estão na pasta DI/Container para rodar basta entrar na pasta public e dar o comando php -S localhost:8090, vc vai precisar do banco de dados, está pensando em desenvolver algo parecido ?? to bem afim de refazer-lo como app em ionic ou android e python com backend, segue um video dele funcionando, https://www.youtube.com/watch?v=Sl1VyR3VGgg

**Comment by Alcala** [comment]
Sim, estava pensando em fazer alguma coisa bem parecida mesmo. Você fez esse como? PHP puro ou algum framework?
